### PR TITLE
Create CI-pipeline.yml

### PR DESCRIPTION
**Pull request by MilenaOlech**
Add CI to validate parse hamnosys script output

### CODE REVIEW

File: .github/workflows/CI-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      # Prepare an environment
+      - name: Environment setup
+        run: |
+          make venv
+
+      # Compare the test and the result files
+      - name: Comparison
+        run: |
+          source .venv/bin/activate
+          python3 parse-hamnosys.py -sf hamnosys_example.txt -df hamnosys_parsed.txt
```
Comment (MilenaOlech) [inline .github/workflows/CI-pipeline.yml]: These files names has been selected exclusively for debug purposes, I'll change them after Marta's PR merge

File: test/test_result_files.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import difflib
+import sys
+import os
+
+test_file = "test/test.txt"
```
Comment (MilenaOlech) [inline test/test_result_files.py]: These files names has been selected exclusively for debug purposes, I'll change them after Marta's PR merge

File: .github/workflows/CI-pipeline.yml (hunk 1 above)
Comment (mplantykow) [reply .github/workflows/CI-pipeline.yml]: PR is merged

File: .github/workflows/CI-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+name: CI-pipeline
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on pull request events for specified branches
+  pull_request:
+    branches: [ pipeline/*, main ]
```
Comment (mplantykow) [inline .github/workflows/CI-pipeline.yml]: Why pipeline?

**Review by mplantykow — APPROVED** [review]

File: .github/workflows/CI-pipeline.yml (hunk 3 above)
Comment (mplantykow) [reply .github/workflows/CI-pipeline.yml]: This got fixed in a following commit, so you can ignore this comment

**Pull request merged**